cub/test/CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
+if(CMAKE_GENERATOR MATCHES "^Visual Studio")
+  if(CUB_ENABLE_RDC_TESTS)
+    if("${CMAKE_VERSION}" VERSION_LESS 3.27.5)
+      # https://gitlab.kitware.com/cmake/cmake/-/merge_requests/8794
+      message(WARNING "CMake 3.27.5 or newer is required to enable RDC tests in Visual Studio.")
```

Comment:
Do you want this to be a hard error? you can call `cmake_minimum_required(VERSION 3.27.5)` here to error out after the message.